frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1584,15 +1580,16 @@ private Map<String, Set<String>> loadFilterMapFromRequest(PortletRequest request
         }
         return filterMap;
     }
-
-    private void addStickyProjectGroupToFilters(PortletRequest request, User user, Map<String, Set<String>> filterMap){
-        String stickyGroupFilter = ProjectPortletUtils.loadStickyProjectGroup(request, user);
-        if (!isNullOrEmpty(stickyGroupFilter)) {
-            String groupFieldName = Project._Fields.BUSINESS_UNIT.getFieldName();
-            filterMap.put(groupFieldName, Sets.newHashSet(stickyGroupFilter));
-            request.setAttribute(groupFieldName, stickyGroupFilter);
-        }
-    }
+// This function was previously called by loadAndStoreStickyProjectGroup in an if block where groupFilterValue==null.
```

Comment:
please remove the commented code.

Reply:
Hi @akapti ,

I have removed it. Please check.